frontend/src/pages/BYONImages/BYONImagesTableRow.tsx

```diff
@@ -52,10 +46,10 @@ const BYONImagesTableRow: React.FC<BYONImagesTableRowProps> = ({
             spaceItems={{ default: 'spaceItemsSm' }}
             alignItems={{ default: 'alignItemsCenter' }}
           >
-            <FlexItem>{obj.display_name}</FlexItem>
-            <FlexItem>
-              <ImageErrorStatus image={obj} />
-            </FlexItem>
+            <ResourceNameTooltip resource={convertBYONImageToK8sResource(obj)}>
+              {obj.display_name}
+            </ResourceNameTooltip>
+            <ImageErrorStatus image={obj} />
           </Flex>
```

Comment:
This seems like you'll not get spacing, why this change?

Reply:
FYI, you're going to run into a conflict -- Gage kept the `FlexItem`s. Perhaps you should match his implementation to avoid conflicts